jaeger_client/senders.py

```diff
@@ -221,6 +226,14 @@ def send(self, batch):
         headers['Content-Length'] = str(len(data))
 
         try:
-            self.client.post(url=self.url, headers=headers, data=data)
+            self._post(data, headers)
         except Exception as e:
             raise_with_value(e, 'POST to jaeger_endpoint failed: {}'.format(e))
+
+
+    def _post(self, data, headers):
+        try:
+            self.client.post(url=self.url, headers=headers, data=data)
+        except ConnectionError:
+            self._reconnect()
```

Comment:
we should log a msg that says the export is being retried.

Reply:
Updated.